data/committee/sortition/sortition_test.go

```diff
@@ -62,3 +68,57 @@ func TestSortitionBasic(t *testing.T) {
 		t.Errorf("wanted %d selections but got %d, d=%d, maxd=%d", expected, hitcount, d, maxd)
 	}
 }
+
+func TestCompareSortitionImpls(tt *testing.T) {
+	partitiontest.PartitionTest(tt)
+
+	// if TOTAL_MONEY env var is set, parse uint64 from env
+	var envTotalMoney uint64
+	if tm := os.Getenv("TOTAL_MONEY"); tm != "" {
+		if val, err := strconv.ParseUint(tm, 10, 64); err == nil {
+			envTotalMoney = val
+			tt.Logf("using TOTAL_MONEY=%d from env", envTotalMoney)
+		}
+	}
+
+	rapid.Check(tt, func(t *rapid.T) {
+		// select one of the protocol committee sizes
+		proto := config.Consensus[protocol.ConsensusCurrentVersion]
+		expectedSize := rapid.OneOf(
+			rapid.Just(proto.NumProposers),
+			rapid.Just(proto.SoftCommitteeSize),
+			rapid.Just(proto.CertCommitteeSize),
+			rapid.Just(proto.NextCommitteeSize),
+			rapid.Just(proto.LateCommitteeSize),
+			rapid.Just(proto.RedoCommitteeSize),
+			rapid.Just(proto.DownCommitteeSize),
+		).Draw(t, "expectedSize").(uint64)
+		//expectedSize := rapid.Uint64Range(1, totalMoney).Draw(t, "expectedSize").(uint64) // draw random
```

Comment:
```suggestion
```

Or if you really need for local testing, it should be placed in the scope of `totalMoney`